cmd/generate/config/rules/nuget.go

```diff
@@ -0,0 +1,27 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func Nuget() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Nuget Password",
+		RuleID:      "nuget-config-password",
+		Regex:       regexp.MustCompile(`(?i)(<add key=\"(ClearTextPassword)|(Password)\"\s*)(value=\"[A-Za-z0-9]+\"\s*/>)`),
```

Comment:
@jankonikola93 Why did you add `Password` here? While not really recommended and only supported on Windows I would say it's OK, because it's encrypted, see https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesourcecredentials.

Reply:
I added it because in my opinion is better to avoid having all kinds of secrets stored in flies on your repo (even encrypted). For me it's better to replace them with placeholders then to store that values on your repo.